### PR TITLE
Doc: Linux troubleshooting with systemd

### PR DESCRIPTION
**Pull request by mrbarletta**
This scenario was happening to me for months, it was due to the hardcoding a DNS resolver in systemd while trying to escape DNS settings from the fortinet vpn.

### CODE REVIEW

**Comment by cicku** [comment]
Systemd will not be the only cause.

**Comment by mrbarletta** [comment]
> Systemd will not be the only cause.

what are you suggesting especifically ?

**Comment by ranbel** [comment]
@dh-cf

File: content/cloudflare-one/faq/teams-troubleshooting.md [hunk 1]
```diff
@@ -179,6 +179,17 @@ If your [Cloudflare Tunnel logs](/cloudflare-one/connections/connect-networks/mo
 
 This error appears if you try to change your [team domain](/cloudflare-one/faq/teams-getting-started-faq/#whats-a-team-domain/team-name) while the [Cloudflare dashboard SSO](/cloudflare-one/applications/configure-apps/dash-sso-apps/) feature is enabled on your account.
 Cloudflare dashboard SSO does not currently support team domain changes. Contact your account team for more details.
+## Warp on Linux shows `DNS connectivity check failed to resolve host="warp-svc."`
+
+This error probably means that you have changed `systemd-resolved` configuration `/etc/systemd/resolved.conf` and set a DNS manually
```
Comment (dh-cf) [inline content/cloudflare-one/faq/teams-troubleshooting.md:184]: 'means that you may have changed'

**Comment by ranbel** [comment]
Thanks for the PR! I've incorporated your feedback into #13096.

**Pull request closed without merging**